app/src/main/java/org/yuttadhammo/BodhiTimer/TimerActivity.kt

```diff
@@ -7,17 +7,13 @@
 
 package org.yuttadhammo.BodhiTimer
 
-import android.content.ActivityNotFoundException
-import android.content.BroadcastReceiver
-import android.content.Context
-import android.content.Intent
-import android.content.IntentFilter
-import android.content.SharedPreferences
+import android.content.*
```

Comment:
The IDE changed this, is the wildcard import a problem?

Reply:
It seems for some people this is important, I don't care either way